opensearch-operator/controllers/dashboardController.go

```diff
@@ -3,96 +3,79 @@ package controllers
 import (
```

Comment:
Can you please make a list of the resources that you moved to configurationController?

Reply:
At the moment this is only the configmap for the `opensearch.yml`